test/cypress/integration/29-host.dashboard.test.js

```diff
@@ -36,6 +36,74 @@ describe('host dashboard', () => {
     });
   });
 
+  describe('unhost collectives', () => {
+    it('unhost collective with zero balance', () => {
+      cy.signup({ redirect: '/brusselstogetherasbl' });
+      const collectiveSlug = randomSlug();
+      cy.get('[data-cy="host-apply-btn"]:visible').click();
+      cy.getByDataCy('host-apply-modal-next').click();
+      cy.getByDataCy('host-apply-collective-picker').click();
+      cy.getByDataCy('host-apply-new-collective-link').click();
+      cy.get(`input[name="name"]`).type('Cavies United');
+      cy.get(`input[name="slug"]`).type(`{selectall}${collectiveSlug}`);
+      cy.get(`input[name="description"]`).type('We will rule the world with our cute squeaks');
+      cy.getByDataCy('checkbox-tos').click();
+      cy.get('button[type="submit"]').click();
+      cy.contains('Cavies United has been created!');
```

Comment:
I suggest looking at `cy.createCollective` for a more performant/less verbose way of creating collectives in cypress tests